src/utils/fim_logger.py

```diff
@@ -288,7 +288,7 @@ def merge_log_files(self, parent_log_output_file, file_prefix, remove_old_files=
     # -------------------------------------------------
     def trace(self, msg):
         # goes to file only, not console
-        level = "TRACE   "  # keeps spacing the same
+        level = "TRACE    "  # keeps spacing the same  (9 chars wide)
```

Comment:
A cleaner way to do this is `level = "TRACE".ljust(9)`

Reply:
Love it. I will duplicate that with the rest of my logging level types